openapi-contracts/src/main/com/sinch/sdk/domains/verification/models/dto/v1/IdentityDto.java

```diff
@@ -71,12 +110,13 @@ public IdentityDto endpoint(String endpoint) {
   }
 
   /**
-   * Value of identity in format defined by its type.
+   * For type &#x60;number&#x60; use an
+   * [E.164](https://community.sinch.com/t5/Glossary/E-164/ta-p/7537)-compatible phone number.
    *
    * @return endpoint
    */
   @JsonProperty(JSON_PROPERTY_ENDPOINT)
-  @JsonInclude(value = JsonInclude.Include.USE_DEFAULTS)
+  @JsonInclude(value = JsonInclude.Include.ALWAYS)
   public String getEndpoint() {
```

Comment:
Should this be a E164 number? I remember you have already a class for it
(If it is the case, it will fail the test `StartVerificationRequestDtoTest`

Reply:
Currently no: generated file is hidden to end user and for network protocol only.
Coming soon: yes. When network protocol won't be anymore hidden but used by end user; this field should be `E164PhoneNumber`